### PR TITLE
chore(Python): added reverse linked list

### PR DESCRIPTION
**Pull request by akshitarora921**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**
Added algorithm to reverse a linked list

**Other information:**

### CODE REVIEW

**Comment by akshitarora921** [comment]
how to add approver @Arsenic-ATG ?

**Comment by akshitarora921** [comment]
The wrong label, It's python code.

**Comment by akshitarora921** [comment]
Error: Process completed with exit code 65.
how to fix this?

File: algorithms/Python/linked_lists/reverse-linkedlist.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    def __init__(self) -> None:
+        self.head = None
+
+     # Function to insert a new node at the beginning
+    def push(self, data):
+        node = Node(data)
+        node.next = self.head
+        self.head = node
+
+     # Function to print the list from begining
```
Comment (Arsenic-ATG) [inline algorithms/Python/linked_lists/reverse-linkedlist.py:21]: There is a typo which due to which `codespell` check is failing.

```suggestion
     # Function to print the list from beginning
```

Comment (akshitarora921) [reply algorithms/Python/linked_lists/reverse-linkedlist.py]: Fixed

**Review by Arsenic-ATG — APPROVED** [review]

**Pull request merged**